config/initializers/inflections.rb

```diff
@@ -11,6 +11,7 @@
 # end
 
 # These inflection rules are supported but not enabled by default:
-# ActiveSupport::Inflector.inflections(:en) do |inflect|
-#   inflect.acronym 'RESTful'
-# end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym 'RESTful'
+  inflect.acronym 'API'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.